### ISSUE TITLE
Official installer is not working with Wineskin on Mac OS

### ISSUE THREAD

**Issue by Sethur** [LOUD-Project/Git-LOUD#55]
I tried installing this on my Intel Mac with Mac OS 12.6.1 using WineSkin. I can successfully run Supreme Commander Forged Alliance with this setup but I wanted to try the LOUD Project as well because of performance issues.

Using the default installer of LOUD did not work at first because it seems to require a 64-bit version of windows, which my WineSkin engine did not support. Changing the WineSkin engine to 64-bit seemed to make the installer run at first, but no window was displayed.

I then tried to clone this repository into the game base folder and run the game in the normal WineSkin 32-bit engine with the suggested command line argument, which worked just fine. However, I no cannot access the official maps, since they are apparently nowhere to be found on official repositories (there is a MAP-Backup repo here, but it does not seem to contain all the necessary maps).

So my question is:
a) Is there an official command line installer that runs for 32-bit environments?
b) Can I get the official maps somewhere else?

PS: Impressive work you have done with this project!

**Comment by binarybubble** [comment]
Hi Sethur

a) no, there is only the Launcher.
b) Join the Discord, I can provide you with the maps that do not come with the distribution.
https://discord.gg/AeNcaYgA

**Comment by Sethur** [comment]
@MostLostNoob Thanks for the quick clarification! I am aware that I could dual-boot on my Mac, but I wanted to avoid the hassle of rebooting for gaming, if possible. Wine worked very well for me for quite a number of games that do not have native binaries.

I'll definitely try the release version you have pointed me to. Time permitting, I'll have a look at the source code of the updater (if it is contained within this repo) and check if it can be easily converted to a CLI tool (using python, bash, etc.).

**Comment by binarybubble** [comment]
> @MostLostNoob Thanks for the quick clarification! I am aware that I could dual-boot on my Mac, but I wanted to avoid the hassle of rebooting for gaming, if possible. Wine worked very well for me for quite a number of games that do not have native binaries.
> 
> I'll definitely try the release version you have pointed me to. Time permitting, I'll have a look at the source code of the updater (if it is contained within this repo) and check if it can be easily converted to a CLI tool (using python, bash, etc.).

https://github.com/rajderks/loud-electron

**Comment by Sethur** [comment]
@binarybubble Thanks for the link. Since the launcher is already an electron app, I can probably just compile and run it natively on Mac OS.

**Comment by Sethur** [comment]
@binarybubble @MostLostNoob Just to let you know: With a little tinkering in the constants.ts file, I got the electron launcher app running under Mac OS. It had modify the paths to refrain from replacing '/' with '\\' and put in a BASE_URI manually, although this might also have worked if I had placed the app in the root of the Supreme Commander folder.

It might be a good idea to not use hardcoded windows style backslashes in the path constants and maybe also include an option in the launcher to actually specify (via a directory select dialogue or by entering the path) the root directory of the game in case the launcher was not placed correctly. This would make the app instantly more portable :-)
